common-util/src/main/java/org/sunbird/common/responsecode/ResponseMessage.java

```diff
@@ -349,6 +349,8 @@
         "Request failed as user access token is not provided.";
     String DUPLICATE_EXTERNAL_IDS =
         "Duplicate external IDs for given idType ({0}) and provider ({1}).";
+    String SYSTEM_ALREADY_INITIALISED = "Request failed as system is already initialised.";
+    String RESTRICTED_REQUEST = "Request is not allowed from current IP address.";
```

Comment:
For security it's preferable that the error message does not indicate that the request can succeed if the IP address is changed. Suggested alternative, 

"Request must be made from an authorised host"